src/components/shared/yamlWidget.js

```diff
@@ -20,7 +24,11 @@ const YamlWidget = (props) => {
         'international_explore_btns': <InternationalExploreButtons />,
         'international_explore_grid': <InternationalExploreGrid />,
         'international_talk_current_student':<InternationalTalkCurrentStudent />,
-        'international_explore_lead':<InternationalExploreLead />
+        'international_explore_lead':<InternationalExploreLead />,
+        'dvm_application_selection':<DVMApplicationSelection />,
```

Comment:
```suggestion
        'dvm_application_selection':<DVMApplication colourOptions={background: "var(--uog-yellow)", colour: "#000000"} icon="fa-circle-3" />,
```

Example of passing variable through to component